test/extension/handlers/topologymutation/handler_test.go

```diff
@@ -304,7 +304,7 @@ func TestHandler_GeneratePatches(t *testing.T) {
 
 			// Expect all response fields to be as expected. responseItems are ignored here and tested below.
 			// Ignore the message to not compare error strings.
-			g.Expect(tt.expectedResponse).To(EqualObject(response, IgnorePaths{{"items"}, {"message"}}))
+			g.Expect(tt.expectedResponse).To(EqualObject(response, IgnorePaths{"items", ".message"}))
```

Comment:
Q: Do you know why items doesn't need a leading dot?

Reply:
AFAIK it's the embedded types which require the `.` prefix - message is part of CommonResponse so it needs a dot, while items is a top level field by itself.